src/services/service.ts

```diff
@@ -225,7 +225,7 @@ export default class Service {
         return `customers/${this.cid}/${resource}`
     }
 
-    protected async serviceCall(call: string, request: any): Promise<any> {
+    protected async serviceCall(call: string, request: any, all_results: boolean = false): Promise<any> {
```

Comment:
I think there is another method somewhere which returns all the data.

Reply:
really? what is it called?